starter-code/css/main.css

```diff
@@ -6,3 +6,148 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.nav-bar {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    align-items: flex-end;
+    
+    
+}
+  
+.nav-bar img {
+    display: flex;
+    justify-content: space-between;
+    float: left;
+    height: 100%;
+    width: 25%;
+    margin-bottom: 30px;
+    margin-top: 20px;
+}
+ul {
+    list-style-type: none;
+}
+  
+.navigation  {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+.navigation li {
+    padding: 30px;
+    color: black;
+    font-size: 2em;
+}
+
+
+.text-block {
+    position: absolute;
+    top: 70%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-size: 30px;
+    text-align: center;
+  }
+
+.main-image img {
+    /* background-image: url(../images/landing.jpg);
+    background-repeat: no-repeat; */
+    height: 900px;
+    width: 100%;
+    
+}
```

Comment:
Line 59-65 aren't needed, line 66-73 take care of the background image. You can also remove the <img>in your index.html on line 26.